### PR TITLE
[connectivity] announce 1.0, deprecate wifi APIs

### PR DESCRIPTION
**Pull request by cyanglaz**
## Description

Bump the version to 1.0.0, announce deprecations to all wifi related APIs

### CODE REVIEW

File: packages/connectivity/connectivity/example/test_driver/integration_test/connectivity_test.dart [hunk 1]
```diff
@@ -22,6 +22,7 @@ void main() {
       expect(result, isNotNull);
       switch (result) {
         case ConnectivityResult.wifi:
+
```
Comment (amirh) [inline packages/connectivity/connectivity/example/test_driver/integration_test/connectivity_test.dart:25]: extra line?

**Review by amirh — APPROVED** [review]
LGTM

File: packages/connectivity/connectivity/lib/connectivity.dart [hunk 2]
```diff
@@ -63,11 +65,15 @@ class Connectivity {
   ///
   /// From Android 8.0 onwards the GPS must be ON (high accuracy)
   /// in order to be able to obtain the BSSID.
+  @Deprecated(
+      'This method is deprecated. Please use wifi_info_flutter instead. See https://github.com/flutter/plugins/blob/master/packages/wifi_info_flutter/wifi_info_flutter/README.md')
```
Comment (ditman) [inline packages/connectivity/connectivity/lib/connectivity.dart:69]: Why not a link to pub.dev? It'd be much shorter!

**Review by ditman — APPROVED** [review]
LGTM! it needs a pass of `dartfmt -w .` however.

File: packages/connectivity/connectivity/lib/connectivity.dart (hunk 2 above)
Comment (cyanglaz) [reply packages/connectivity/connectivity/lib/connectivity.dart:69]: wifi_info_flutter is not published yet :(

File: packages/connectivity/connectivity/example/test_driver/integration_test/connectivity_test.dart (hunk 1 above)
Comment (cyanglaz) [reply packages/connectivity/connectivity/example/test_driver/integration_test/connectivity_test.dart]: done

**Pull request merged**